backend/api/rooms/[roomId]/delete.js

```diff
@@ -0,0 +1,45 @@
+const handler = require('../../../handler/handler');
+const Room = require('../../../handler/Classes/Room');
+
+const allowCors = fn => async (req, res) => {
+    res.setHeader('Access-Control-Allow-Credentials', true);
+    res.setHeader('Access-Control-Allow-Origin', '*');
+    res.setHeader('Access-Control-Allow-Methods', 'POST,OPTIONS');
+    res.setHeader(
+        'Access-Control-Allow-Headers',
+        'X-CSRF-Token, X-Requested-With, Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, X-Api-Version'
+    );
+    if (req.method === 'OPTIONS') {
+        res.status(200).end();
+        return;
+    }
+    return await fn(req, res);
+};
+
+const deleteF = async (req, res) => {
```

Comment:
```suggestion
const delete = async (req, res) => {
```

Reply:
delete is a special word in js cant use it as variable

Reply:
i changed it to 'del'